_includes/install/jquery.md

```diff
@@ -1,10 +1,10 @@
 ### Step 1: Download the jQuery package
 
-We have a jQuery plugin [available here](http://archive.tinymce.com/download/custom_package.php) if you'd rather install TinyMCE using this method.
+We have a jQuery plugin [available here]({{site.gettiny}}custom-builds/) if you'd rather install {{site.productname}} using this method.
```

Comment:
This page is only for v4, so not sure if you want to remove it for now?

Reply:
That page seems wrong and horrible... and now it's gone.